test/test_torch.py

```diff
@@ -12303,6 +12309,36 @@ def non_zero_rand(size, dtype, device):
                     non_zero_rand((2, 2), dtype=dtype, device=device),
                     non_zero_rand((2, 2), dtype=dtype, device=device))
 
+    # This function tests that a nan value is returned for input values not in domain
+    @dtypes(torch.float32, torch.complex64)
+    def test_acosh_domain(self, device):
```

Comment:
When using the @dtypes decorator your signature needs take the arguments `self, device, dtype`, the dtype argument will be populated with the dtype value (in this case, torch.float32 or torch.complex64). Then you can construct tensors with dtype=dtype.

Reply:
Oh, I missed that? Sorry!!

Reply:
Solved, thanks for pointing this out @mruberry! (sorry for not checking this before)